NebulaPatcher/Patches/Transpilers/PlanetFactory_Transpiler.cs

```diff
@@ -73,13 +72,19 @@ static IEnumerable<CodeInstruction> OnBeltBuilt_Transpiler(ILGenerator gen, IEnu
                 }
             }
 
+            if(!found)
```

Comment:
Debugging code - this should be removed before merge - if it is required, then please make it more explanatory so the player can understand what this error means